form-flow/src/main/kotlin/com/ritense/formflow/domain/instance/FormFlowStepInstance.kt

```diff
@@ -79,6 +79,9 @@ data class FormFlowStepInstance(
                 "key" to stepKey,
                 "submissionData" to instance.getSubmissionDataContext()
             ),
+            "instance" to mapOf(
```

Comment:
Maybe just a variable at the lowest level, so instanceId? Additional properties are also on this same level.